### ISSUE TITLE
New sensor model: AMA Mercury Thermometer (wet-bulb)

### ISSUE THREAD

**Issue by MKrieger29** [OceanOPS/sot-metadata#33]
Adding of a new sensor model in the OceanOPS database
AMA_MERCURY THERMOMETER as a wet-bulb. 
(A record is already existing as a dry-bulb)

Requestor: Joel Cabrie

**Comment by anthoninlize** [comment]
@jbcohuet we assigned you to validate the entry, since you worked a lot on this. We will implement it then.

**Comment by jbcohuet** [comment]
yes that's fine, you can add it.
But I don't see any wet bulb temperature sensor in the SOT list. 
I thought we worked on a first list. DO you remember that ?

**Comment by MKrieger29** [comment]
In order to create this new sensor model I propose to 
- Rename AMA_MERCURY THERMOMETER to **AMA_MERCURY DRY BULB THERMOMETER**
- Create the new sensor model **AMA_MERCURY WET BULB THERMOMETER**, with the sensor_type THERMOMETER_MERCURY_WET_BULB. 
Do you agree with this?

**Comment by jbcohuet** [comment]
In found a previsous verison of ref table with the following entires
![image](https://user-images.githubusercontent.com/103053433/195034211-3a12a5b5-24df-41f8-9f23-ed87d51a11a5.png)

**Comment by anthoninlize** [comment]
> In found a previsous verison of ref table with the following entires ![image](https://user-images.githubusercontent.com/103053433/195034211-3a12a5b5-24df-41f8-9f23-ed87d51a11a5.png)

Yes, this probably is not relevant anymore since psychrometers (in that case) have been placed under psychrometer sensor types (several exist), not wet bulb.
See green tags there:
![image](https://user-images.githubusercontent.com/17119571/195035268-9a3b7411-2add-411d-bb62-ef3f53049f3c.png)

**Comment by jbcohuet** [comment]
Can we keep exactly the same reference for AMA but only adding only the wet bulb parameter ?
It is the same sensor, in other conditions

Otherwise we should rename all the mercury thermometers ?

**Comment by jbcohuet** [comment]
Ok forget my remark about the psychrometer. Psychrometer is wet bulb + dry bulb, so we need wet bulb sensor only.

**Comment by jbcohuet** [comment]
Maybe we have to create also a generic wet bulb sensor

**Comment by anthoninlize** [comment]
We need to be consistent and rational about it. 
Segregating wet and dry bulb has been done for some sensors. It has not for others. We're still at the beginning so we can change things rather easily.

Is it worth having 2 sensors ? Or should it be an external parameter ? (eg SENSOR TYPE, which can have 2 sensor types for 1 sensor model, and then select the correct one).
We can have a quick call to chat about it too.
